src/components/filters/DateTimeFilter.vue

```diff
@@ -99,32 +97,61 @@ export default Vue.extend({
       const date = this.dateRange
       if (!date.startDate || !date.endDate) return 'Select...'
       if (date.startDate.toISOString() === date.endDate.toISOString()) {
-        return date.startDate.toLocaleString()
+        return this.beautifyDateTime(date.startDate)
       } else {
-        return `${date.startDate.toLocaleString()} - ${date.endDate.toLocaleString()}`
+        return `${this.beautifyDateTime(date.startDate)} - ${this.beautifyDateTime(date.endDate)}`
       }
+    },
+    pickerFormat () {
+      if (this.time) return 'yyyy-mm-dd HH:MM:ss'
+      else return 'yyyy-mm-dd'
     }
   },
   watch: {
     value (newValue) {
-      this.dateRange = newValue
+      this.dateRange.startDate = this.createDateFromValue(this.value[0])
+      this.dateRange.endDate = this.createDateFromValue(this.value[1])
```

Comment:
Waarom niet 1 functie call, bijv. `updateDateRange(this.value)`; scheelt 3 regels in watch en beforeMount

Reply:
thats nicer, i made it setDateRange

Reply:
Vergeet niet om de value parameter te gebruiken in de nieuwe functie